tests/tests-core/src/main/java/com/gentics/mesh/core/language/LanguageEndpointTest.java

```diff
@@ -0,0 +1,368 @@
+package com.gentics.mesh.core.language;
+
+import static com.gentics.mesh.test.ClientHelper.call;
+import static com.gentics.mesh.test.ElasticsearchTestMode.TRACKING;
+import static com.gentics.mesh.test.TestDataProvider.PROJECT_NAME;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Random;
+
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import com.gentics.mesh.FieldUtil;
+import com.gentics.mesh.core.data.perm.InternalPermission;
+import com.gentics.mesh.core.rest.error.GenericRestException;
+import com.gentics.mesh.core.rest.lang.LanguageListResponse;
+import com.gentics.mesh.core.rest.lang.LanguageResponse;
+import com.gentics.mesh.core.rest.node.NodeCreateRequest;
+import com.gentics.mesh.core.rest.node.NodeListResponse;
+import com.gentics.mesh.core.rest.node.NodeResponse;
+import com.gentics.mesh.core.rest.project.ProjectResponse;
+import com.gentics.mesh.parameter.client.GenericParametersImpl;
+import com.gentics.mesh.parameter.impl.NodeParametersImpl;
+import com.gentics.mesh.parameter.impl.ProjectLoadParametersImpl;
+import com.gentics.mesh.test.MeshTestSetting;
+import com.gentics.mesh.test.TestSize;
+import com.gentics.mesh.test.context.AbstractMeshTest;
+import com.gentics.mesh.test.definition.BasicRestTestcases;
+
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.reactivex.Observable;
+
+@MeshTestSetting(elasticsearch = TRACKING, testSize = TestSize.FULL, startServer = true)
+public class LanguageEndpointTest extends AbstractMeshTest implements BasicRestTestcases {
```

Comment:
Even if creation/updating and deleting languages over the REST API is not supported, we should still add tests for the correct behaviour when the endpoints are called.

Reply:
I'll just comment those out, as per proposal below.